components/Alert/AlertNotification.tsx

```diff
@@ -0,0 +1,38 @@
+import { Alert as AlertDefault, AlertTitle, AlertDescription } from './Alert';
+import { CheckCircle, XCircle, Info, AlertTriangle } from 'lucide-react';
+
+interface AlertProps {
+  variant?: 'success' | 'error' | 'warning';
+  message: string;
+}
+
+const variantConfig = {
+  success: {
+    title: 'Success!',
+    icon: CheckCircle,
+  },
+  error: {
+    title: 'Error!',
+    icon: XCircle,
+  },
+  default: {
+    title: 'Info',
+    icon: Info,
+  },
+  warning: {
+    title: 'Warning!',
+    icon: AlertTriangle,
+  },
+};
+
+export default function Alert({ variant, message }: AlertProps) {
```

Comment:
You'll thank me later, but this need to be a Functional component. Also, need to add unit tests for this component.